### PR TITLE
DEPRECATED [incubator/haproxy-ingress] deprecation notice

### PR DESCRIPTION
**Pull request by jcmoraisjr**
#### Special notes for your reviewer:

This is a deprecation notice with the link to the new repository.

#### Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/haproxy-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 name: haproxy-ingress
-version: 0.0.27
+version: 0.0.28
 appVersion: 0.7.2
 home: https://github.com/jcmoraisjr/haproxy-ingress
-description: Ingress controller implementation for haproxy loadbalancer.
+description: DEPRECATED! See https://github.com/haproxy-ingress/charts
```
Comment (davidkarlsen) [inline incubator/haproxy-ingress/Chart.yaml]: See https://github.com/helm/charts/blob/master/PROCESSES.md for required changes when deprecating a chart.

Comment (jcmoraisjr) [reply incubator/haproxy-ingress/Chart.yaml]: Missed that, thanks for the review. Updated.

**Review by scottrigby — CHANGES_REQUESTED** [review]
Also requires a note in NOTES.txt

**Comment by jcmoraisjr** [comment]
Now NOTES.txt also updated, thanks!

**Pull request merged**